app/views/characters/_character_summary.html.erb

```diff
@@ -0,0 +1,217 @@
+<div class="mt-5">
```

Comment:
Watch out for your indentation. Make sure your edit has a tab of two instead of four